cypress/integration/tests/administrator/credentials/field_validation.test.ts

```diff
@@ -0,0 +1,21 @@
+import { hasToBeSkipped, login } from "../../../../utils/utils";
+import { CredentialsSourceControlUsername } from "../../../models/administrator/credentials/credentialsSourceControlUsername";
+import { getRandomCredentialsData } from "../../../../utils/data_utils";
+import { CredentialType } from "../../../types/constants";
+import { Credentials } from "../../../models/administrator/credentials/credentials";
+
+describe("Credentials filter validations", { tags: "@tier2" }, function () {
+    const cred = new CredentialsSourceControlUsername(
+        getRandomCredentialsData(CredentialType.proxy)
+    );
+    before("Login and Create Test Data", function () {
+        // Prevent hook from running, if the tag is excluded from run
+        if (hasToBeSkipped("@tier2")) return;
+
+        // Perform login
+        login();
+    });
+    it("Validate fields for too short (2 symbols) and too long (120+ symbols) length ", () => {
```

Comment:
Is it only this test in this file or we are planning to add more ?

Reply:
I will add more cause I will want to validate explicitly all creds types where this validation will be applicable

Reply:
Ok :+1: